Sources/PackageGraph/PackageRegistryRelease.swift

```diff
@@ -0,0 +1,58 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import struct Foundation.URL
+import struct TSCUtility.Version
+
+public struct PackageRegistryRelease {
+    /// Information from the signing certificate.
+    public enum Certificate {
+        case trusted(commonName: String, organization: String, identity: String?)
+        case untrusted(commonName: String, organization: String)
+        case none
```

Comment:
what is the difference between trusted and untrusted?

Reply:
Trusted according to configured or default root certs is my understanding.

Reply:
`valid` better than `trusted`? @andyp-apple wdyt?

Reply:
Both `trusted` and `untrusted` are valid, right?